katalog/etcd-backup-s3/cronjob.yaml

```
@@ -0,0 +1,110 @@
+# Copyright (c) 2017-present SIGHUP s.r.l All rights reserved.
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file.
+
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: etcd-backup-s3
```

Let's set some standard labels, for example:
```suggestion
  name: etcd-backup-s3
  labels:
    app.kubernetes.io/name: etcd-backup-s3
    app.kubernetes.io/instance: etcd-backup-s3-dr
    app.kubernetes.io/version: "1.0.0"
    app.kubernetes.io/component: disaster-recovery
    app.kubernetes.io/part-of: KFD
```